src/Model/benefit.js

```diff
@@ -0,0 +1,79 @@
+class Benefit {
+	getBenefit(visitDate, menu, beforeDiscountPrice) {
+		if (beforeDiscountPrice < 10000)
+			return {
+				DdayMoney: 0,
+				weekdayOrweekendMoney: 0,
+				starMoney: 0,
+				totalBenefit: 0,
+				dateIndex: 0,
+			}
+		const dateIndex = this.#getDayOfWeek(visitDate)
+		const DdayMoney = this.#getD_Day(visitDate)
+
+		const weekdayOrweekendMoney =
```

Comment:
```suggestion
		const weekdayOrWeekendMoney =
```
중간에 카멜케이스가 빠진 것 같군요!